app/assets/stylesheets/stops_show_styles.css

```diff
@@ -0,0 +1,42 @@
+.errors{
+  font-weight: bold;
+}
+
+.name{
+  font-size: 6em;
+  padding-left:15px;
+}
+
+.errors{
+  padding-left:15px;
+}
```

Comment:
you have .errors on line 1 and line 10, you should combine the styling here since it's for the same class